### PR TITLE
Release 1.0

### PR DESCRIPTION
**Pull request by ricsantos**
Brings ManUp inline with [ionic-manup](https://github.com/NextFaze/ionic-manup) by:

- updating the config.json format, now has (optional) nested `ios` object, and newer, friendlier keys `latest`, `minimum`, `url`, etc... #15 #11 
- add support for the `enabled` key, aka maintenance mode #16

### CODE REVIEW

File: ManUp/ManUp.m [hunk 1]
```diff
@@ -78,6 +79,7 @@ - (id)init {
 	if (self = [super init]) {
 		self.updateInProgress = NO;
```
Comment (swoolcock) [inline ManUp/ManUp.m:50]: Tabs D: D: D:

File: ManUp/ManUp.m [hunk 2]
```diff
@@ -265,10 +282,31 @@ - (void)checkAppVersion {
...
         
     } else {
+        BOOL enabled = [self settingForKey:kManUpConfigAppIsEnabled] ? [[self settingForKey:kManUpConfigAppIsEnabled] boolValue] : YES;
+        if (enabled && self.maintenanceModeShown) {
+            self.maintenanceModeShown = NO;
+            [self.alertController dismissViewControllerAnimated:YES completion:nil];
+            self.alertController = nil;
+            
+        } else if (!enabled && !self.maintenanceModeShown) {
+            [self showAlertWithTitle:[NSString stringWithFormat:NSLocalizedString(@"%@ Unavailable", nil), appName]
```
Comment (swoolcock) [inline ManUp/ManUp.m]: Comments in `NSLocalizedString` please. 💩

File: ManUp/ManUp.m [hunk 3]
```diff
@@ -265,10 +282,31 @@ - (void)checkAppVersion {
     NSComparisonResult minVersionComparisonResult = [ManUp compareVersion:installedVersion toVersion:minVersion];
     NSComparisonResult currentVersionComparisonResult = [ManUp compareVersion:installedVersion toVersion:currentVersion];
     
-    if (self.alertController) {
+    if (self.alertController && !self.maintenanceModeShown) {
```
Comment (awulf) [inline ManUp/ManUp.m]: Maintenance mode should be shown in favour of an update alert. That is an update alert should be removed snd replaced with a maintenance alert

File: ManUp/ManUp.m [hunk 4]
```diff
@@ -265,10 +282,31 @@ - (void)checkAppVersion {
...
     } else {
+        BOOL enabled = [self settingForKey:kManUpConfigAppIsEnabled] ? [[self settingForKey:kManUpConfigAppIsEnabled] boolValue] : YES;
+        if (enabled && self.maintenanceModeShown) {
+            self.maintenanceModeShown = NO;
+            [self.alertController dismissViewControllerAnimated:YES completion:nil];
+            self.alertController = nil;
+            
+        } else if (!enabled && !self.maintenanceModeShown) {
+            [self showAlertWithTitle:[NSString stringWithFormat:NSLocalizedString(@"%@ Unavailable", nil), appName]
+                             message:[NSString stringWithFormat:NSLocalizedString(@"%@ is currently unavailable. Please check back later.", nil), appName]
```
Comment (awulf) [inline ManUp/ManUp.m]: Add comments to localized strings where parameters are given eg "{appName} Unavailable"

File: ManUp/ManUp.m (hunk 3 above)
Comment (ricsantos) [reply ManUp/ManUp.m]: Good point. I will change it so that it always removes a current alert before showing a new one.

File: ManUp/ManUp.m (hunk 1 above)
Comment (ricsantos) [reply ManUp/ManUp.m:50]: 😱

File: ManUp/ManUp.m (hunk 2 above)
Comment (ricsantos) [reply ManUp/ManUp.m]: Will fix 👍

File: ManUp/ManUp.m (hunk 4 above)
Comment (ricsantos) [reply ManUp/ManUp.m]: Will fix 💃

**Review by ricsantos — COMMENTED** [review]
I didn't mean to start a review here.

**Pull request merged**